psi4/src/psi4/libmints/wavefunction.h

```diff
@@ -375,9 +375,9 @@ class PSI_API Wavefunction : public std::enable_shared_from_this<Wavefunction> {
     static void initialize_singletons();
 
     /// Returns the DOCC per irrep array. Not recommended for unrestricted code.
-    const Dimension doccpi(bool assume_socc_alpha = true) const;
+    const Dimension doccpi(bool warn_on_beta_socc = true) const;
     /// Returns the SOCC per irrep array. Not recommended for unrestricted code.
```

Comment:
Shouldn't the argument be documented here? e.g. `warn_on_beta_socc: trigger warning on singly occupied beta orbitals that break assumptions made in the old-fashioned DOCC/SOCC handling`